tee-worker/service/src/cli.yml

```diff
@@ -155,6 +155,22 @@ subcommands:
                 long: request-state
                 short: r
                 help: Run the worker and request key and state provisioning from another worker.
+            - teeracle-interval:
+                required: false
+                long: teeracle-interval
+                short: i
+                help: Set the teeracle exchange rate update interval. Example of accepted syntax <5 seconds 15 minutes 2 hours 1 days> or short <5s15m2h1d>
+                takes_value: true
+            - reregister-teeracle-interval:
+                required: false
+                long: reregister
+                help: Set the teeracle reregistration interval. Example of accepted syntax <5 seconds 15 minutes 2 hours 1 days> or short <5s15m2h1d>
+                takes_value: true
```

Comment:
I think this should be removed actually, see https://github.com/litentry/litentry-parachain/pull/2488/files#diff-6acbe3078fcde879295ce6a6b23acd22ba163f837b3fde7700b268a03ec3930b

Reply:
good catch, it seems I've resolved conflict in a wrong way. Thanks